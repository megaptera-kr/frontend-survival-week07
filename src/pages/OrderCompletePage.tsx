export default function OrderCompletePage() {
  return (
    <section>
      <h1>주문 완료, 서둘러 배달가겠습니다!</h1>
    </section>
  );
}
