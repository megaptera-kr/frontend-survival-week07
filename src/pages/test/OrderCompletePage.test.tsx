import { render } from '@testing-library/react';
import { Route } from 'react-router-dom';

import OrderCompletePage from '../OrderCompletePage';
import { withRouter } from '../../tests/helper';

describe('Order Complete page', () => {
  it('render correctly', () => {
    const { asFragment } = render(withRouter(<Route path="/" element={<OrderCompletePage />} />));

    expect(asFragment()).toMatchSnapshot();
  });
});
