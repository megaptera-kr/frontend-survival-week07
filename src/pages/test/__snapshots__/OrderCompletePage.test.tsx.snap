// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Order Complete page render correctly 1`] = `
<DocumentFragment>
  <section>
    <h1>
      주문 완료, 서둘러 배달가겠습니다!
    </h1>
  </section>
</DocumentFragment>
`;
